src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -43,8 +44,10 @@ public void InvokeNonDurabalitySupportError(object sender, string message, Excep
             var handler = OnNonDurabalitySupportError;
             if (handler == null)
             {
-                throw new InvalidDataException(message + Environment.NewLine +
-                     "An exception has been thrown because there isn't a listener to the OnNonDurabalitySupportError event on the storage options.", e);
+                if (_log.IsInfoEnabled)
```

Comment:
No, it _should_ throw if no one is actually listening. This is a problem that we will handle in `ServerStore`